src/components/Metadata/Metadata.js

```diff
@@ -114,29 +101,23 @@ const Metadata = props => {
                         props.onEdit(selectedRange.id))(range)}
                       blackAndWhiteMode={props.blackAndWhiteMode}
                     />
-                  </Meta>
                 )
               )}
-            </PoseGroup>
           </div>
           <div className="metadata__content">
-            <PoseGroup animateOnMount={true}>
               {markers.map(marker => {
                 return (
-                  <Meta key={marker.id}>
                     <MarkerMetadata
                       inset={rangesToShow.length}
                       key={marker.id}
-                      highlight={marker === currentMarker}
+                      highlight={false}
```

Comment:
Is there another style for highlighting that we could use, it seemed quite useful to have some visual indicator of which marker you were looking at

Reply:
Maybe @joncameron's [request below](https://github.com/digirati-co-uk/timeliner/pull/295#issuecomment-482618526) is sufficient to disambiguate markers and a highlight isn't needed?